cmake/Modules/FindMKLDNN.cmake

```diff
@@ -43,6 +43,25 @@ LIST(APPEND MKLDNN_INCLUDE_DIR ${IDEEP_INCLUDE_DIR})
 IF(MKL_FOUND)
   LIST(APPEND MKLDNN_LIBRARIES ${MKL_LIBRARIES})
   LIST(APPEND MKLDNN_INCLUDE_DIR ${MKL_INCLUDE_DIR})
+  # The OMP-related variables of MKL-DNN have to be overwritten here,
+  # if MKL is using, and the OMP version is defined by MKL.
+  # MKL_LIBRARIES_xxxx_LIBRARY is defined by MKL.
+  # INTEL_MKL_DIR gives the MKL root path.
+  IF (INTEL_MKL_DIR)
+    SET(MKLROOT ${INTEL_MKL_DIR})
+    IF(WIN32)
+      SET(MKLIOMP5DLL ${MKL_LIBRARIES_libiomp5md_LIBRARY} CACHE STRING "Overwrite MKL-DNN omp dependency" FORCE)
+    ELSE(WIN32)
+      IF (MKL_LIBRARIES_gomp_LIBRARY)
```

Comment:
Where is this flag set?

Reply:
MKL_LIBRARIES_gomp_LIBRARY is set by FindMKL.cmake